code/modules/mob/living/carbon/superior_animal/human/subtype/solelite/solelite

```diff
@@ -0,0 +1,100 @@
+	wander = 0
+	maxHealth = 300
+	health = 300
+
+	armor = list(melee = 55, bullet = 55, energy = 70, bomb = 80, bio = 90, rad = 25) //Legitmently their armor (melee is higher {45} to account for AI stupidness)
+
+	//range/ammo stuff
+	ranged = 1
+	rapid = 1
+	rapid_fire_shooting_amount = 5
+	ranged_cooldown = 2
+	projectiletype = /obj/item/projectile/bullet/magnum_40/hv
+	projectilesound = 'sound/weapons/guns/fire/smg_fire.ogg'
+	limited_ammo = TRUE
+	mag_drop = TRUE
+	rounds_left = 20
+	mag_type = /obj/item/ammo_magazine/smg_magnum_40/empty
+	mags_left = 4 //4+1
+
+	melee_damage_lower = 20
+	melee_damage_upper = 30
+	breath_required_type = 0 // Doesn't need to breath, in a hardsuit
+	breath_poison_type = 0 // Can't be poisoned
+	min_air_pressure = 0 // Doesn't need pressure
+	attacktext = "slashed"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	meat_amount = 0
+	meat_type = null
+	leather_amount = 0
+	bones_amount = 0
+//They are all wearing suits
+	breath_required_type = NONE
+	breath_poison_type = NONE
+	min_breath_required_type = 0
+	min_breath_poison_type = 0
+
+	min_air_pressure = 0
+	min_bodytemperature = 0
+
+//Drops
+	meat_amount = 1
+	meat_type = /obj/item/reagent_containers/food/snacks/meat/human
+
+	inherent_mutations = list(MUTATION_HEART, MUTATION_LUNG, MUTATION_LIVER, MUTATION_BLOOD_VESSEL, MUTATION_MUSCLES, MUTATION_NERVES)
+
+	faction = "elitemerc"
+
+/mob/living/carbon/superior_animal/human/solelite/handle_breath(datum/gas_mixture/breath) //we have are own air supplies
+	return
+
+/mob/living/carbon/superior_animal/human/solelite/handle_environment(var/datum/gas_mixture/environment) //are armor legit is a hardsuit
+	return
+
+/mob/living/carbon/superior_animal/human/solelite/start_pulling(var/atom/movable/AM)
+	to_chat(src, SPAN_WARNING("Your hand gets pushed away from \the [src]. !"))
+	return
+
+/mob/living/carbon/superior_animal/human/solelite/melee
+	icon_state = "syndicate_stormtrooper_melee"
+	desc = "Black and charred hardsuit, sword and shield to match, a being of absolute terror to those who dare approach."
+	armor = list(melee = 65, bullet = 65, energy = 80, bomb = 80, bio = 90, rad = 25) //Melee shield
+	ranged = 0
+	melee_damage_lower = 40
+	melee_damage_upper = 55
+
+
+/mob/living/carbon/superior_animal/human/solelite/pistol
+	desc = "Black and charred hardsuit, revolver in-hand, a being of absolute terror to those who dare approach"
+	icon_state = "syndicate_stormtrooper_pistol"
+	projectiletype = /obj/item/projectile/bullet/rifle_75/hv
+	drop_items = list(/obj/item/gun/projectile/revolver/hornet)
+	projectilesound = 'sound/weapons/guns/fire/12mm_revolver.ogg'
+	rounds_left = 6
+	mag_type = /obj/item/ammo_magazine/speed_loader_kurtz_50/empty
+	mags_left = 5 //5+1
+
+/mob/living/carbon/superior_animal/human/solelite/shotgunner
+	desc = "Black and charred hardsuit, omnirifle shotgun in-hand, a being of absolute terror to those who dare approach."
+	icon_state = "syndicate_stormtrooper_shotgun"
```

Comment:
This comment applies to all the sprites, honestly - but it might be good to kitbash the wielded sprite of the weapons we want them to have with the sprite they use - these sprites predate Eris iirc and are the old ss13 style of 'guns are only held in one hand always.'